package.json

```
@@ -76,7 +76,7 @@
     "lint": "standard | snazzy",
     "lint:fix": "standard --fix | snazzy",
     "test": "node scripts/generate-pem && npm run test:tap && npm run test:node-fetch && npm run test:fetch && npm run test:cookies && npm run test:wpt && npm run test:websocket && npm run test:jest && npm run test:typescript",
-    "test:cookies": "node scripts/verifyVersion 16 || tap test/cookie/*.js",
+    "test:cookies": "node scripts/verifyVersion 16 || node --test test/cookie/*.js",
```

Hey, I noticed that using `node --test` causes nyc to get stuck.   
However, removing the `--test` flag appears to resolve the issue. 

```suggestion
    "test:cookies": "node scripts/verifyVersion 16 || node test/cookie/*.js",
```

---

`"test:cookies": "node scripts/verifyVersion 16 || borp --coverage test/cookie/*.js",` using `borp`

---

As of now, I would avoid the usage of the `--coverage` parameter, in order to avoid low coverage errors during the migration. wdyt?